### PR TITLE
Updates on layout

### PR DESCRIPTION
**Pull request by ThomWensink**
Some layout and color changes. @oliverds @neo22s This is a dark layout and more Wordpress like. If you guys dont like it we can reverse this commit :)

### CODE REVIEW

**Comment by oliverds** [comment]
It feels too dark to me. I like more a light header.

File: themes/default/views/oc-panel/main.php [hunk 1]
```diff
@@ -62,10 +62,12 @@
           <div class="col-xs-12">
             <?=View::factory('oc-panel/sidebar',array('user'=>$user))?>
             <div id="page-wrapper">
+              <div class="row">
```
Comment (oliverds) [inline themes/default/views/oc-panel/main.php]: Right! This row div was missing.

**Comment by ThomWensink** [comment]
@oliverds Personally I prefer the dark theme because of the improved readability, and also to indicate the difference between menu and content. 

 Maybe we should give the user an option for light or dark theme? Guess that isnt so hard to create?

**Comment by oliverds** [comment]
I know this is a perception thing but for me it's easier to read black text on a white background.

So let's have a light and dark color skin for the panel. It seems there are few variables to change so should be easy.

**Comment by oliverds** [comment]
Also, after this layout update I'm seeing theses issues:

* 
screen shot 2016-04-17 at 8 25 51 pm


* 
screen shot 2016-04-17 at 8 28 59 pm

**Comment by ThomWensink** [comment]
@oliverds I solved these with the last commit. 

The room between the header and the menu was done on purpose to seperate header menu from sidemenu but I have removed the space as you requested.

**Pull request merged**